untitled.js

```diff
@@ -0,0 +1,14 @@
+var name = 'Christian'
+let age = 15
+const gender = 'male' 
+let grade = 9;
+let height = 5,9;
```

Comment:
This isn't a valid value. It will throw an error. You can represent this as total inches instead.